scripts/watermarkstat

```diff
@@ -177,28 +174,46 @@ class Watermarkstat(object):
 
         return pg_index
 
+    def build_header(self, wm_type):
+        if wm_type is None:
+            print >> sys.stderr, "Header info is not available!"
+            sys.exit(1)
+
+        header_map = wm_type["obj_map"]
+        single_key = header_map.keys()[0]
+        header_len = len(header_map[single_key])
+        min_idx = float("inf")
+
+        for name, counter_oid in header_map[single_key].items():
+            curr_idx = int(wm_type["idx_func"](counter_oid))
+            min_idx = min(min_idx, curr_idx)
+
+        self.min_idx = int(min_idx)
+        self.header_list = ["{}{}".format(wm_type["header_prefix"], idx) for idx in range(self.min_idx, self.min_idx + header_len)]
```

Comment:
With this change, we lose the "Port" header for the first column. Is it possible for you to add it here?

Reply:
Done!

Reply:
@neethajohn: Thanks! can you please also update the sample output in the PR description?